docs/DOCKER.md

```diff
@@ -112,6 +112,47 @@ Using VSCODE
 -----------------------
 In order to use VSCODE in debugging mode with DOCKER, check out [VSCODE_CONFIGURATION](VSCODE_CONFIGURATION.md)
 
+## Working on anyway-etl and Airflow
+
+Anyway ETL processes and Airflow server are developed in a different repository: [hasadna/anyway-etl](https://github.com/hasadna/anyway-etl)
+but we use the same Docker Compose environment.
+
+For some tasks you will also need to set secret values in the `.env` file, ask a team member for these values.
+
+**Running anyway-etl tasks**
+
+* Pull the latest anyway-etl Docker image: `docker-compose pull anyway-etl`
+* See the available anyway-etl commands: `docker-compose run anyway-etl --help`
+* Run a command: `docker-compose run anyway-etl cbs parse-all`
+
+**Developing anyway-etl tasks**
+
+To develop anyway-etl using the Docker Compose environment you first need to clone [hasadna/anyway-etl](https://github.com/hasadna/anyway-etl).
+The clone should be a sibling directory to anyway, so it will be at `../anyway-etl` relative to anyway repository.
+
+* Build the anyway-etl Docker image: `docker-compose -f docker-compose.yml -f ../anyway-etl/docker-compose-override.yaml build anyway-etl`
+* Run anyway-etl commands: `docker-compose -f docker-compose.yml -f ../anyway-etl/docker-compose-override.yaml run anyway-etl --help`
+    * When running this command, any changes you make to anyway or anyway-etl code will take effect immediately without requireing to rebuild the image. 
+
+**Running the Airflow server**
+
+* Pull the airflow server image: `docker-compose pull airflow-webserver airflow-scheduler`
```

Comment:
I get the following errors when running this command - is this ok or should something be fixed?

```WARNING: The IMAP_MAIL_USER variable is not set. Defaulting to a blank string.
WARNING: The IMAP_MAIL_PASSWORD variable is not set. Defaulting to a blank string.
Pulling airflow-webserver ... error
Pulling airflow-scheduler ... error

ERROR: for airflow-webserver  Head https://docker.pkg.github.com/v2/hasadna/anyway-etl/anyway-etl-airflow/manifests/latest: no basic auth credentials

ERROR: for airflow-scheduler  Head https://docker.pkg.github.com/v2/hasadna/anyway-etl/anyway-etl-airflow/manifests/latest: no basic auth credentials
ERROR: Head https://docker.pkg.github.com/v2/hasadna/anyway-etl/anyway-etl-airflow/manifests/latest: no basic auth credentials
Head https://docker.pkg.github.com/v2/hasadna/anyway-etl/anyway-etl-airflow/manifests/latest: no basic auth credentials```

Reply:
fixed